next.clayui.com/content/docs/css/components/stickers.md

```diff
@@ -0,0 +1,381 @@
+---
+title: "Stickers"
+description: "Stickers are a visual way to quickly identify content in a different way than badges and labels."
```

Comment:
I'd probably leave off "in a different way than badges and labels" because it assumes the user knows about badges and labels and also doesnt actually really give any more clarity as to what it does. Should be fine with just "Stickers are a visual way to quickly identify content"

Reply:
Thanks for the tip, I just copied from lexicon's website